src/App.tsx

```diff
@@ -1,24 +1,171 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import Footer from './components/Footer';
+import TodoList from './components/TodoList';
 
-const USER_ID = 0;
+export const userId = 10590;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filterType, setFilterType] = useState('all');
```

Comment:
Do not rely on the unknown string, make enum for this.
```suggestion
  const [filterType, setFilterType] = useState(FilterType.ALL);
```